src/contents/events/python-bootcamp/index.md

```diff
@@ -5,8 +5,19 @@ date: 2018-12-09
 location: seminar hall
 cover: './cover.png'
 ---
-## Event details
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
 
-## ANything in Markdown
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
\ No newline at end of file
+Python Bootcamp was a series of Python-workshops which was held for interested students of Sahyadri.
+The main initiative of this series of workshops was to basically train the attendees/participants from
+beginner to advanced level in python.
+This workshop was mainly headed by Abhi Gatty and other members of SOSC.
+
```

Comment:
Would be great if we had all the members who conducted the workshop being mentioned.